core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
Do we need `toString` method in this helper reader? And I just realised that I have no idea how `toString` should look like for complex readers like this:

```scala
for { 
  a <- RequiredParam("a")
  b <- OptionalParam("b")
} yield (a, b.getOrElse("foo"))
```
I would expect to see something like `RequiredParam 'a' & OptionalParam 'b'` or similar. But I have no idea how to achieve that.